plugins/org.jkiss.dbeaver.ext.generic/plugin.xml

```diff
@@ -1355,6 +1355,62 @@
                     <file type="license" path="drivers/apache_lic/LICENSE.txt" bundle="drivers.salesforce"/>
                 </driver>
 
+                <!-- Teiid -->
+                <driver
+                        id="teiid"
+                        label="Teiid"
+		                icon="platform:/plugin/org.jkiss.dbeaver.model/icons/connection/database_icon.png"
+                        iconBig="platform:/plugin/org.jkiss.dbeaver.model/icons/connection/database_icon.png"
+                        class="org.teiid.jdbc.TeiidDriver"
+                        sampleURL="jdbc:teiid:{database}@mms://{host}:{port}"
+                        defaultPort="31000"
+                        webURL="https://jdbc.teiid.org/"
+                        description="Teiid JDBC Driver.">
+
+                    <file type="jar" path="maven:/org.teiid:teiid:jdbc:LATEST" bundle="!drivers.teiid" custom="true"/>
+                    <file type="jar" path="drivers/teiid" bundle="drivers.teiid"/>
+                    <file type="license" path="drivers/teiid/LICENSE.txt" bundle="drivers.teiid"/>
```

Comment:
Please add the source for license downloading. For example

`<file type="license" path="https://www.apache.org/licenses/LICENSE-2.0.txt" bundle="!drivers.teiid"/>`